src/tests/components/Course.test.js

```diff
@@ -4,6 +4,7 @@ import { Course } from "../../components/Course";
 describe("Course", () => {
     it("displays the course title", () => {
         const course = {
+            length: "",
             title: "Mathematics",
             liveUrl: "",
         };
```

Comment:
```suggestion
        const course = {
            title: "Mathematics",
            liveUrl: "",
        };
```
I don't think we need to have the length specified in this test - it might confuse people in future

Reply:
When I'd summitted the code initially there was a failure in testing. This was one of the areas identified. I have just re run format and fc just to cover both off and all the tests and it works with the line removed so I'll take your suggestion as you are correct.